web-module/src/main/webapp/resources/structEditJS/dateTime/celements-date-time-field.js

```diff
@@ -292,203 +296,195 @@
+        this.shadowRoot.insertBefore(this.#datePickerIcon, this.datePart.nextSibling);
+        if (this.hasTime()) {
+          this.#timePickerIcon = this.#timePickerIcon ?? new Element('i', {
+            'title': 'Time Picker',
+            'class': 'CelTimePicker timeInputField halflings halflings-time'
+          });
+          this.shadowRoot.insertBefore(this.#timePickerIcon, this.timePart.nextSibling);  
         }
       }
 
-      _updateValueAttribute() {
-        const _me = this;
-        console.debug('_updateValueAttribute: ', _me._hiddenInputElem, _me.getAttribute('value'));
-        if (_me._hiddenInputElem) {
-          _me._hiddenInputElem.setAttribute('value', _me.getAttribute('value'));
-        }
+      #addHiddenInput() {
+        this.parentElement.insertBefore(this.#hiddenInputElem, this);
       }
 
       connectedCallback() {
-        const _me = this;
-        console.log('DateTimeFiled connectedCallback: ', _me.isConnected, _me.parentElement);
-        if (!_me._hiddenInputElem) {
-          _me._hiddenInputElem = new Element('input', {
-            'type': 'hidden',
-            'name': _me.getAttribute('name'),
-            'value': _me.value
-          });
-        }
-        _me.parentElement.insertBefore(_me._hiddenInputElem, _me);
-        if (!_me._dateTimeFieldControler) {
-          _me._dateTimeFieldControler = new CELEMENTS.structEdit.DateTimeInputHandler(_me);
+        console.debug('connectedCallback', this.isConnected, this.hasTime(), this);
+        this.#addHiddenInput();
+        this.#addInputFields();
+        this.#addPickerIcons();
+        if (!this.#dateTimeFieldController) {
+          this.#dateTimeFieldController = new CELEMENTS.structEdit.DateTimeInputHandler(this);
         }
       }
 
       disconnectedCallback() {
-        const _me = this;
-        console.log('DateTimeFiled disconnectedCallback: ', _me.isConnected, _me.parentElement,
-          _me._hiddenInputElem);
-        if (!_me._hiddenInputElem) {
-          _me._hiddenInputElem.remove();
-        }
+        console.debug('disconnectedCallback', this.isConnected, this);
+        this.#hiddenInputElem.remove();
       }
 
       static get observedAttributes() {
-        return ['name', 'value'];
+        return ['name', 'value', 'placeholder-date', 'placeholder-time'];
       }
 
-      attributeChangedCallback(attrName, oldValue, newValue) {
-        const _me = this;
-        switch (attrName) {
-          case "name":
-            _me._updateNameAttribute();
+      attributeChangedCallback(name, oldValue, newValue) {
+        console.debug('attributeChangedCallback', this, name, newValue);
+        switch (name) {
+          case 'name':
+          case 'value':
+            this.#hiddenInputElem.setAttribute(name, newValue);
             break;
-          case "value":
-            _me._updateValueAttribute();
+          case 'placeholder-date':
+            this.datePart.setAttribute('placeholder', newValue);

```

Comment:
I think a break is missing here. Otherwise changing a placeholder-date always changes placeholder-time too.